.github/actions/terraform-stats/README.md

```diff
@@ -0,0 +1,110 @@
+# Terraform Stats
+
+[![Test action](https://github.com/JosiahSiegel/terraform-stats/actions/workflows/test_action.yml/badge.svg)](https://github.com/JosiahSiegel/terraform-stats/actions/workflows/test_action.yml)
+
+## Synopsis
+
+Output the following statistics for the Terraform environment:
+1. Terraform version
+2. Drift count
+   * "Drift" refers to changes made outside of Terraform and does not necessary match any resources listed for changes.
+3. Resource drifts
+4. Change count
+   * "Change" refers to change actions that Terraform plans to use to move from the prior state to a new state.
+5. Change percent
+   * Percentage of changes to total resources.
+6. Resource changes
+
+## Usage
+
+```yml
+- name: Terraform stats
+  uses: josiahsiegel/terraform-stats@<latest-version>
```

Comment:
needs update

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.